commands/builds/create.js

```diff
@@ -81,9 +88,10 @@ module.exports = {
   help: 'Create build from contents of current dir',
   description: 'create build',
   flags: [
-    { name: 'source-url', description: 'Source URL that points to the tarball of your application\'s source code', hasValue: true },
-    { name: 'tar', description: 'Path to the executable GNU tar', hasValue: true },
-    { name: 'version', description: 'Description of your new build', hasValue: true }
+    { name: 'source-url', description: 'source URL that points to the tarball of your application\'s source code', hasValue: true },
+    { name: 'tar', description: 'path to the executable GNU tar', hasValue: true },
+    { name: 'version', description: 'description of your new build', hasValue: true },
+    { name: 'include-vcs-ignore', description: 'include files ignores by VCS (.gitignore, ...) from the build' }
```

Comment:
my original complaint about this one was because it was a string flag that was acting as a boolean, but now that I'm looking at the implementation it seems that osx doesn't support this flag at all. Given that over half of our users use osx, I feel it would be better to reverse this flag otherwise osx users will get an error when using this, right?